### ISSUE TITLE
Window position 'Additional Information' 'User Rules' 'Quick Backlist' ... not stored

### ISSUE THREAD

**Issue by AndiB-2** [bitwiseworks/InJoy#10]
Double click on a line in Dropped Packets pops up the Additional Info window on a fixed position. Should remember last position at least when positioned with 'Shft'. Same with other windows.

**Comment by SilvanScherrer** [comment]
this is right now a restriction of the used dynamic window classes. we need to change them to latest version or rewrite the gui with qt.
